### PR TITLE
Business critical blog + doc updates

### PR DESCRIPTION
**Pull request by meagancojocar**
update after initial business critical push
- fixed docs links to self hosted installers
- updated self hosted screenshot in blog post
- updated self hosted components page, api page and console page
- added trial link to self hosted docs

### CODE REVIEW

**Comment by meagancojocar** [comment]
@susanev the svgs on mobile don't look great but the pngs do. Any advice? My instinct would be to screenshot the svg so they are clear but guessing you have more professional methods!

**Comment by meagancojocar** [comment]
@cnunciato could you double check the links I added on the blog post feature list before we merge? Much appreciated!

**Comment by susanev** [comment]
@meagancojocar are you using safari? if so the issue is not us, safari does not render svgs well at all in general (its a known issue). if we are super worried about it we could detect that and display pngs just for them but i don't think we should make that happen right now for this.

File: themes/default/content/blog/business-critical-launch/index.md [hunk 1]
```diff
@@ -50,10 +50,10 @@ Full details of our Pulumi Service editions can be found on [our pricing page]({
 
 The Pulumi Business Critical edition includes everything in the Pulumi Enterprise Edition, plus:
 
-- Organization policies and enforcement
-- Self-hosting available
-- Cross-domain identity (SCIM)
-- Automatic export of audit logs
+- [Organization policies and enforcement]({{< relref "/docs/guides/crossguard/" >}})
+- [Self-hosting available]({{< relref "/docs/guides/crossguard/" >}})
+- [Cross-domain identity (SCIM)]({{< relref "/docs/guides/scim/" >}})
```
Comment (susanev) [inline themes/default/content/blog/business-critical-launch/index.md:55]:
```suggestion
- [Organization policies and enforcement]({{< relref "/docs/guides/crossguard" >}})
- [Self-hosting available]({{< relref "/docs/guides/crossguard" >}})
- [Cross-domain identity (SCIM)]({{< relref "/docs/guides/scim" >}})
```

**Review by susanev — APPROVED** [review]

**Comment by susanev** [comment]
why is this not running the test suite?

**Comment by susanev** [comment]
@meagancojocar for the future, no trailing slashes on links (this is a hugo thing)

File: themes/default/content/blog/business-critical-launch/index.md [hunk 2]
```diff
@@ -50,10 +50,10 @@ Full details of our Pulumi Service editions can be found on [our pricing page]({
...
 The Pulumi Business Critical edition includes everything in the Pulumi Enterprise Edition, plus:
 
-- Organization policies and enforcement
-- Self-hosting available
-- Cross-domain identity (SCIM)
-- Automatic export of audit logs
+- [Organization policies and enforcement]({{< relref "/docs/guides/crossguard" >}})
+- [Self-hosting available]({{< relref "/docs/guides/crossguard" >}})
+- [Cross-domain identity (SCIM)]({{< relref "/docs/guides/scim" >}})
+- [Automatic export of audit logs]({{< relref "docs/intro/pulumi-service/audit-logs/#automated-export" >}})
```
Comment (susanev) [inline themes/default/content/blog/business-critical-launch/index.md:56]:
```suggestion
- [Automatic export of audit logs]({{< relref "docs/intro/pulumi-service/audit-logs#automated-export" >}})
```

**Pull request merged**